examples/platform/linux/CommissioneeShellCommands.cpp

```diff
@@ -84,6 +88,27 @@ static CHIP_ERROR CommissioneeHandler(int argc, char ** argv)
         return error  = SendUDC(true, chip::Transport::PeerAddress::UDP(commissioner, port));
     }
 #endif // CHIP_DEVICE_CONFIG_ENABLE_COMMISSIONER_DISCOVERY_CLIENT
+    else if (strcmp(argv[0], "restartmdns") == 0)
+    {
+        // if commissioning mode is true
+        if ((argc > 1) && (strcmp(argv[1], "true") == 0))
```

Comment:
Since this is only  3 values, the pair "false true" is invalid.

How about making argv[1]  one of "disabled, enabled, enabled_with_additional" or similar naming?